tests/integration_tests/performance/test_boottime.py

```diff
@@ -11,6 +11,7 @@
 
 # The maximum acceptable boot time in us.
 MAX_BOOT_TIME_US = 150000
+MAX_BOOT_TIME_INITRD_US = 400000
```

Comment:
Hmm, I was expecting the difference to be smaller between with initrd_path and without. I wonder if that difference is expected?

Reply:
That value is definitely too high. From the last test I get:

```
integration_tests/performance/test_boottime.py 
Boot time with no network is: 99778 us
...Boot time with network configured is: 118604 us
.Boot time with initrd + network configured is: 148323 us
.Boot time with initrd is: 150110 us
```

- initrd tests a.t.m. are always run with a network enabled, so there's the 20ms overhead from that,
- the remaining 30ms should be due to initrd specific code (checking for cmpression method, mounting the rootfs partition,...).

Reply:
Okay, so basically we could safely use 160000 us instead?

Reply:
If you think 10ms accounts for the boot time variance, then yes!